packages/gatsby-cli/src/init-starter.ts

```diff
@@ -29,26 +28,13 @@ const spawn = (
   const [file, ...args] = cmd.split(/\s+/)
   return spawnWithArgs(file, args, options)
 }
-// Checks the existence of yarn package and user preference if it exists
+// Checks the existence of yarn package
 // We use yarnpkg instead of yarn to avoid conflict with Hadoop yarn
 // Refer to https://github.com/yarnpkg/yarn/issues/673
-const shouldUseYarn = async (): Promise<boolean> => {
+const checkForYarn = async (): Promise<boolean> => {
   try {
     execSync(`yarnpkg --version`, { stdio: `ignore` })
-
-    let packageManager = getPackageManager()
-    if (!packageManager) {
-      // if package manager is not set:
-      //  - prompt user to pick package manager if in interactive console
-      //  - default to yarn if not in interactive console
-      if (isTTY()) {
-        packageManager = (await promptPackageManager()) || `yarn`
```

Comment:
I think this is only use of `promptPackageManager` - with it not being used anymore - let's also remove `promptPackageManager` (dead code) completely from https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/src/util/package-manager.ts ?